### DISCUSSION TITLE
vdh not responding

### DISCUSSION THREAD

**Discussion by sgksm85** [aclap-dev/video-downloadhelper#989] [Help and Q&A]
### Your question

When I click on the extension, there's no response at all. Since other extensions work normally, it's only this particular extension that's affected.

### Addon & CoApp version and details

When I click on the extension, there's no response at all. Since other extensions work normally, it's only this particular extension that's affected.

**Comment by paulrouget** [comment]
Ah. Seems bad. Can you try this: https://github.com/aclap-dev/video-downloadhelper/wiki/How-to-enable-logs
